cmd/agent/install_script.sh

```diff
@@ -34,53 +34,54 @@ trap "rm -f $npipe" EXIT
 
 function on_error() {
     printf "\033[31m$ERROR_MESSAGE
-It looks like you hit an issue when trying to install the Agent.
+It looks like you hit an issue when trying to install the StackState Agent v2.
 
 Troubleshooting and basic usage information for the Agent are available at:
 
-    https://docs.datadoghq.com/agent/basic_agent_usage/
+    https://docs.stackstate.com/guides/basic_agent_usage/
 
-If you're still having problems, please send an email to support@datadoghq.com
-with the contents of ddagent-install.log and we'll do our very best to help you
+If you're still having problems, please send an email to info@stackstate.com
+with the contents of $logfile and we'll do our very best to help you
 solve your problem.\n\033[0m\n"
 }
 trap on_error ERR
 
-if [ -n "$DD_HOSTNAME" ]; then
-    dd_hostname=$DD_HOSTNAME
+if [ -n "$STS_HOSTNAME" ]; then
+    sts_hostname=$STS_HOSTNAME
 fi
 
-if [ -n "$DD_API_KEY" ]; then
-    apikey=$DD_API_KEY
+if [ -n "$STS_API_KEY" ]; then
+    api_key=$STS_API_KEY
 fi
 
 no_start=
-if [ -n "$DD_INSTALL_ONLY" ]; then
+if [ -n "$STS_INSTALL_ONLY" ]; then
     no_start=true
 fi
 
 # comma-separated list of tags
-if [ -n "$DD_HOST_TAGS" ]; then
-    host_tags=$DD_HOST_TAGS
+if [ -n "$STS_HOST_TAGS" ]; then
+    host_tags=$STS_HOST_TAGS
+fi
+
+if [ -n "$STS_URL" ]; then
+    sts_url=$STS_URL
 fi
 
-if [ -n "$DD_URL" ]; then
-  dd_url=$DD_URL
+if [ -n "$CODE_NAME" ]; then
+    code_name=$CODE_NAME
 else
-  dd_url="datadoghq.com"
+    code_name="stable"
 fi
 
-dd_upgrade=
-if [ -n "$DD_UPGRADE" ]; then
-  dd_upgrade=$DD_UPGRADE
+if [ ! $api_key ]; then
+    printf "\033[31mAPI key not available in STS_API_KEY environment variable.\033[0m\n"
+    exit 1;
 fi
 
-if [ ! $apikey ]; then
-  # if it's an upgrade, then we will use the transition script
-  if [ ! $dd_upgrade ]; then
-    printf "\033[31mAPI key not available in DD_API_KEY environment variable.\033[0m\n"
+if [ ! $sts_url ]; then
+    printf "\033[31mStackState url not available in STS_URL environment variable.\033[0m\n"
     exit 1;
-  fi
 fi
 
 # OS/Distro Detection
```

Comment:
Same here as below, fo now we only support ubuntu, so we should present the user an error on other osses